src/docs/guides/add-admin-menu/README.md

```diff
@@ -115,9 +115,9 @@ using OrchardCore.Navigation;
 
 namespace MyModule
 {
-    public class AdminMenu : INavigationProvider
+    public sealed class AdminMenu : INavigationProvider
     {
-        private readonly IStringLocalizer S;
+        internal readonly IStringLocalizer S;
```

Comment:
Why all these `S` visibility changes? We don't need them more visible than `private`.

Reply:
Reason we changed them a while ago go from protected to private is to avoid the analyzer warning. Now that the files are sealed, we can't use protected. Using internal will still be internal to our project and will not have the warning. The problem with the language extractor is that it is looking for S. So using internal is the best option here to avoid warnings